lib/migration.js

```diff
@@ -783,8 +783,7 @@ function mixinMigration(PostgreSQL) {
           if (identical && siKeys.length > 1) {
             if (siKeys.length !== i.keys.length) {
               // lengths differ, obviously non-matching
-              // ??? orderMatched is not defined and not used anywhere else
-              // orderMatched = false;
+              identical = false;
```

Comment:
This change looks unrelated to the other change you are making above. What scenario are you trying to fix with this change?